src/couch_replicator/src/couch_replicator_worker.erl

```diff
@@ -31,6 +31,9 @@
 -define(BULK_GET_RATIO_THRESHOLD, 0.5).
 -define(BULK_GET_RATIO_DECAY, 0.25).
 -define(BULK_GET_RETRY_SEC, 37).
+-define(REVS_DIFF_RATIO_THRESHOLD, 0.95).
+-define(REVS_DIFF_RATIO_DECAY, 0.4).
+-define(REVS_DIFF_RETRY_SEC, 37).
```

Comment:
Good idea. Add some comments. The types "threshold", "decay" and "retry_sec" are similar between both the revs_diff and bulk_get

Reply:
Updated with a test and updated the description of the coefficients.

The tests set up two scenarios, one with an empty target, and then check that less than the expected number of revs_diff calls are made (more than half a skipped), the other populates the target with the same docs and we check that the expected number (docs_count / replication_batch_size) are made.

Because the batch size and number of revs_diff is not necessarily deterministic, opted to use thresholds (> half, < max etc as opposed to exact comparisons).